src/librustdoc/clean/mod.rs

```diff
@@ -2163,7 +2163,8 @@ fn clean_use_statement(
         return Vec::new();
     }
 
-    let (doc_meta_item, please_inline) = import.attrs.lists(sym::doc).get_word_attr(sym::inline);
+    let doc_meta_item = import.attrs.lists(sym::doc).get_word_attr(sym::inline);
```

Comment:
Let's name this after the attribute, not after the AST structure.
```suggestion
    let inline_attr = import.attrs.lists(sym::doc).get_word_attr(sym::inline);
```